sdk/identity/azure-identity/azure/identity/_credentials/managed_identity.py

```diff
@@ -150,7 +150,8 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
                 self._endpoint_available = False
 
         if not self._endpoint_available:
-            raise CredentialUnavailableError(message="IMDS endpoint unavailable")
+            raise CredentialUnavailableError(message="ManagedIdentityCredential authentication unavailable. "
```

Comment:
Is this valid python code to have a string on a new line like this?

Reply:
fixed formatting, placed whole string in a second line